test/unit/src/objects/LOD.tests.js

```diff
@@ -78,9 +78,9 @@ export default QUnit.module( 'Objects', () => {
 			lod.addLevel( low, 50 );
```

Comment:
You could add a test for hysteresis in this by adding one similar to the `getObjectForDistance` test but where you stay around the hysteresis range and make sure no change happens until you get out of it

Reply:
Good point - will add a unit test if this ends up being merged 👍